src/electionguard/election.cpp

```diff
@@ -159,6 +159,9 @@ namespace electionguard
 
         auto cryptoExtendedBaseHash = hash_elems({cryptoBaseHash.get(), commitmentHash.get()});
 
+        // ensure the elgamal public key instance is set as a fixed base
+        elGamalPublicKey->setIsFixedBase(true);
```

Comment:
when setting the fixed base flag, the exponentiation functions know that they should use a table to compute the exponentiation and not just do it directly.